source/switch/main.cpp

```diff
@@ -551,8 +555,12 @@ void threadFunc(void *args) {
 
 		double endTime = (double)svcGetSystemTick() * SECONDS_PER_TICKS;
 
-		if (endTime - startTime < TARGET_FRAMETIME && !(inputTransferKeysHeld & KEY_ZL)) {
-			svcSleepThread((u64)fabs((TARGET_FRAMETIME - (endTime - startTime)) * 1000000000 - 100));
+		if (endTime - startTime < TARGET_FRAMETIME) {
+			if (switchRLButtons &&!(inputTransferKeysHeld & KEY_R)){
```

Comment:
please simplify the logic to something like `inputTransferKeysHeld & (switchRLButtons ? KEY_R : KEY_ZR)`